### PR TITLE
Avoid using already canceled context for mqsend

### PR DESCRIPTION
**Pull request by fishy**

### CODE REVIEW

File: events/events.go [hunk 1]
```diff
@@ -55,6 +55,8 @@ type Config struct {
 	//
 	// If the passed in context object already has an earlier deadline set,
 	// that deadline will be respected instead.
+	// But if the passed in context is already canceld,
```
Comment (pacejackson) [inline events/events.go:58]: Small typo, should be “ cancelled”

File: events/events.go [hunk 2]
```diff
@@ -103,6 +105,11 @@ func (q *Queue) Close() error {
 
 // Put serializes and puts an event into the event queue.
 func (q *Queue) Put(ctx context.Context, event thrift.TStruct) error {
+	if ctx.Err() != nil {
+		// The request context is already canceled,
```
Comment (pacejackson) [inline events/events.go:109]: Small typo, should be “ cancelled”

File: events/events.go (hunk 1 above)
Comment (fishy) [reply events/events.go:58]: The variable is called `context.Canceled`: https://pkg.go.dev/context?tab=doc#pkg-variables

Comment (fishy) [reply events/events.go:58]: technically both `cancelled` and `canceled` are correct in English, I think.

Comment (pacejackson) [reply events/events.go:58]: Ya, I guess they are...

:til:

(I’ve probably written it both ways...)

Comment (pacejackson) [reply events/events.go:58]: Well, _this_ one still has a small type 😅

“canceld”

Comment (fishy) [reply events/events.go]: :shakes-fist: fixed.

**Pull request merged**